src/main/scala/rps/Game.scala

```diff
@@ -2,69 +2,64 @@ package rps
 
 import scala.io.StdIn.readLine
 import scala.util.Random
+import rps.models._
+import rps.models.Move.Paper
+import rps.models.Move.Rock
+import rps.models.Move.Scissors
```

Comment:
```suggestion
import rps.models.Move.{Paper, Rock, Scissors}
```
volendo si possono raggruppare così, rimangono forse un po' più leggibili

Reply:
Sì, deve averlo fatto in automatico il formattatore o l'auto import, mi è completamente sfuggito.